lxd/loki/loki.go

```diff
@@ -330,11 +330,19 @@ func (c *Client) HandleEvent(event api.Event) {
 			}
 		}
 
+		keys := make([]string, len(context))
+
+		for k := range context {
+			keys = append(keys, k)
+		}
+
+		sort.Strings(keys)
+
 		messagePrefix := ""
 
-		// Add the remaining context as the message prefix.
-		for k, v := range context {
-			messagePrefix += fmt.Sprintf("%s=\"%s\" ", k, v)
+		// Add the remaining context as the message prefix. The keys are sorted alphabetically.
+		for _, k := range keys {
+			messagePrefix += fmt.Sprintf("%s=\"%s\" ", k, context[k])
```

Comment:
if you use backticks here you wouldnt need to escape the double quotes, it might make it slightly easier to read.

Reply:
I think `%q` would be even better.

Reply:
indeed! :)